overlord/snapstate/snapstate_test.go

```diff
@@ -72,7 +72,7 @@ type snapmgrTestSuite struct {
 }
 
 func (s *snapmgrTestSuite) settle() {
-	err := s.o.Settle(5 * time.Second)
+	err := s.o.Settle(30 * time.Second)
```

Comment:
this seems a bit excessive, I rechecked, do you remember which test had the problem actually?

afaict from local runs only the recent  TestInstallWithoutCoreTwoSnapsRunThrough is problematic